dockerfiles/scripts/install_cmake.sh

```diff
@@ -3,5 +3,5 @@ mkdir -p /tmp/src
 cd /tmp/src
 
 echo "Installing cmake"
-curl https://github.com/Kitware/CMake/releases/download/v3.27.1/cmake-3.27.1-linux-`uname -m`.tar.gz -sSL --retry 5  -o /tmp/src/cmake-3.27.1-linux-`uname -m`.tar.gz
-tar -zxf /tmp/src/cmake-3.27.1-linux-`uname -m`.tar.gz --strip=1 -C /usr
+curl https://github.com/Kitware/CMake/releases/download/v3.27.2/cmake-3.27.2-linux-`uname -m`.tar.gz -sSL --retry 5  -o /tmp/src/cmake-3.27.2-linux-`uname -m`.tar.gz
```

Comment:
```suggestion
CMAKE_VERSION="3.27.2"
curl https://github.com/Kitware/CMake/releases/download/v${CMAKE_VERSION}/cmake-${CMAKE_VERSION}-linux-`uname -m`.tar.gz -sSL --retry 5  -o /tmp/src/cmake-${CMAKE_VERSION}-linux-`uname -m`.tar.gz
```

Reply:
Declaring a bash variable to hold the version and use it everywhere will save a lot of copy&paste in the future :)

Reply:
Thanks for your great suggestion. I will do it in the next PR. 
I need to merge this PR first to bring nightly pipelines back to normal.